save-demo/src/main/kotlin/com/saveourtool/save/demo/storage/ToolStorage.kt

```diff
@@ -22,5 +22,5 @@ class ToolStorage(
         return ToolKey(ownerName, toolName, vcsTagName, executableName)
     }
 
-    override fun buildPathToContent(rootDir: Path, key: ToolKey): Path = rootDir / key.ownerName / key.toolName / key.vcsTagName / key.executableName
+    override fun buildPathToContent(rootDir: Path, key: ToolKey): Path = rootDir / key.organizationName / key.projectName / key.version / key.fileName
```

Comment:
suggestion to store keys in database and use primary ID as path. It avoids problem with renaming and escaping a spec symbols in file name

Reply:
Will be done when the frontend part of file uploading is done